### PR TITLE
Additional test cases

### PR DESCRIPTION
**Pull request by Psirex**
PR adds the following tests:
- unit tests for `OssifiableProxy`
- scenario test for incomplete exited keys reporting

### CODE REVIEW

File: test/0.4.24/lido-deposit-scenarios.test.js [hunk 1]
```diff
@@ -4,22 +4,44 @@ const { EvmSnapshot, setBalance, getBalance } = require('../helpers/blockchain')
 const { ZERO_ADDRESS } = require('../helpers/constants')
 const { assert } = require('../helpers/assert')
 const { wei } = require('../helpers/wei')
-const { StakingModuleStub } = require('../helpers/stubs/staking-module.stub')
 const { PUBKEY_LENGTH, FakeValidatorKeys, SIGNATURE_LENGTH } = require('../helpers/signing-keys')
-const { GenericStub } = require('../helpers/stubs/generic.stub')
+const { ContractStub } = require('../helpers/contract-stub')
```
Comment (TheDZhon) [inline test/0.4.24/lido-deposit-scenarios.test.js:8]: The file is missing in this PR :eyes:

**Review by TheDZhon — COMMENTED** [review]
:+1: thank you for delivering this

Can't find one test helper :detective:

File: test/0.4.24/lido-deposit-scenarios.test.js (hunk 1 above)
Comment (Psirex) [reply test/0.4.24/lido-deposit-scenarios.test.js:8]: Oops, forgot to include it in the initial commit, sorry. Added it

**Review by TheDZhon — APPROVED** [review]
:+1:

**Pull request merged**